src/store/apiSlice.ts

```diff
@@ -53,12 +54,13 @@ type StoreResourceMap<AttributeType> = Record<string, StoreResource<AttributeTyp
 
 // The list of potential resource types. IMPORTANT: When a new resource type is integrated, it must
 // be added to this list.
-export const RESOURCES = ["logins", "organisations", "users"] as const;
+export const RESOURCES = ["logins", "organisations", "users", "delayedJobs"] as const;
 
 type ApiResources = {
   logins: StoreResourceMap<LoginDto>;
   organisations: StoreResourceMap<OrganisationDto>;
   users: StoreResourceMap<UserDto>;
+  delayedJobs: StoreResourceMap<DelayedJobDto>;
```

Comment:
:art: a minor thing, but I think keeping this in alphabetical order is sort of nice.

Reply:
sorted